test/tests/interfaces/pbs_preempt_params.py

```diff
@@ -216,7 +197,7 @@ def test_set_unset_preempt_order(self):
         else:
             self.fail(msg)
 
```

Comment:
I think we can remove line 170 to 199 as this we already doing in  line 168: self.common_tests(param, msg)

Reply:
right - will remove these lines.